test/Serialization/Inputs/macro_decl.swift

```diff
@@ -0,0 +1,6 @@
+@resultBuilder
+public struct Builder {
+  static func buildBlock(_: Int...) -> Void {}
+}
+@freestanding(expression)
+public macro macroWithBuilderArgs(@Builder _: () -> Void) = #externalMacro(module: "A", type: "B")
```

Comment:
There's already an input file with macro declarations for serialization tests in `Serialization/Inputs/def_macros.swift` (and macro implementations for serialization tests in `Serialization/Inputs/def_macro_plugin.swift`). Can we keep the macro serialization inputs in one place?

Reply:
Good catch. Done. However for some reason `Serialization/Inputs/def_macros.swift` requires macosx, and we should look into that later.

Reply:
Yeah, all of the macro tests do, because the Swift Parser is not enabled on Linux CI yet.